README.md

```diff
@@ -4,29 +4,127 @@
 
 This is a Node module that makes it very simple
 to interact with MySQL databases.
-It has the same API as https://github.com/mvolkmann/postgresql-easy.
+Fundamentally, it wraps Connection and Pool objects from [mysql](https://www.npmjs.com/package/mysql) and adds convenience methods for common tasks and interaction patterns.
+It was inspired by [postgres-easy](https://github.com/mvolkmann/postgresql-easy).
 
-To install this, run `npm install -S mysql-easier`
+Migrating from 1.x to 2.x?  Please see: [Migrating](#migrating-from-1x-to-2x)
 
-## Setup
+## Installation
+
+To install mysql-easier, run `npm install -S mysql-easier`
+
+## General Use
 
 ```js
-const MySqlConnection = require('mysql-easier');
-const mysql = new MySqlConnection(config);
+const mySqlEasier = require('mysql-easier');
+
+// Configure the global pool
+mySqlEasier.configure(config);
+
+// Get a connection from the pool
+const myConn = await mySqlEasier.getConnection();
+
+// Do some database stuff
+
+const newId = await myConn.insert('food', {name: 'Orange', color: 'Orange'});
```

Comment:
Minor feedback, but maybe using a food where the name and color differ (like 'apple' and 'red') would be more clear.

Reply:
Heh.  Agreed.